tests/parsing/GlobbingTest.py

```diff
@@ -218,7 +219,8 @@ def setUp(self):
     def _test_glob(self, pattern, file_list):
         results = sorted([os.path.normcase(g) for g in glob(pattern)])
         file_list = sorted([os.path.normcase(f) for f in file_list])
-        self.assertEqual([i for i in results if "__pycache__" not in i],
+        self.assertEqual([i for i in results
+                          if re.search("(__pycache__|\.pyc)", i) is None],
```

Comment:
you removed the `r` prefix for the string, you need to keep that in this case (as you do `\.`, without the `r` prefix you need to do `\\.`)

same below